paper-slider.html

```diff
@@ -292,7 +292,7 @@
      * @method increment
      */
     increment: function() {
-      this.value = this._clampValue(this.value + this.step);
+      this.value = this._clampValue(this.value + this.step).toString();
```

Comment:
Isn't `value` a `Number`?

https://github.com/PolymerElements/iron-range-behavior/blob/master/iron-range-behavior.html#L28-L33

Or is it being overridden to `String` at some point?